crates/sncast/src/starknet_commands/account/mod.rs

```diff
@@ -21,6 +21,8 @@ pub mod deploy;
 pub mod import;
 pub mod list;
 
+type NestedMap<T> = HashMap<String, HashMap<String, T>>;
```

Comment:
I moved this up here since it's also used in the `read_and_flatten` function `list.rs` and `generate_account_name` function in `import.rs`. Thus, preventing duplicity.